cypress/commons/actions/generic/Scenarios.js

```diff
@@ -243,14 +243,35 @@ function createScenario(scenarioName, isMaster, datasetOrMasterName, runTemplate
   });
 }
 
-function validateScenario() {
-  return getScenarioValidateButton().click();
+function validateScenario(scenarioId) {
+  const validateScenarioAlias = api.interceptUpdateScenario(scenarioId);
+  const getScenarioAlias = api.interceptGetScenario(scenarioId);
+
+  getScenarioValidateButton().click();
+  Scenarios.getScenarioValidationStatusLoadingSpinner().should('be.visible');
+
+  api.waitAlias(getScenarioAlias);
```

Comment:
I think the wait for `getScenarioAlias` should be after the wait for `validateScenarioAlias` (idem in other functions below)